.github/workflows/on_PR_meson.yaml

```diff
@@ -126,6 +127,36 @@ jobs:
           meson setup "${{github.workspace}}/build" -Dauto_features=${{matrix.deps}} -Dwarning_level=3 -Dcpp_std=c++20
           meson compile -C "${{github.workspace}}/build" --verbose
           meson test -C "${{github.workspace}}/build" --verbose
+  Cygwin:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        deps: ['enabled', 'disabled']
+        platform: ['x86', 'x86_64']
```

Comment:
32-bit support was dropped by Cygwin some time ago.

Reply:
Didn't know that. I'll drop.